tools/combine_metaphlan2_humann2/combine_metaphlan2_humann2.xml

```diff
@@ -47,6 +47,18 @@
     </outputs>
 
     <tests>
+    	<test>
+    		 <param name="metaphlan2_file" value="metaphlan2_file.txt"/>
+    		 <param name="humann2_file" value="humann2_file.txt"/>
+    		 <param name='type' value="gene_families"/>
+    		 <output name="gene_families_output_file" value="combined_humann2_metaphlan2.txt"/>
+    	</test>
+    	<test>
+    		 <param name="metaphlan2_file" value="metaphlan2_file.txt"/>
```

Comment:
Input files are not identical in both tests. First test is supposed to get gene family abundance file as `humann2_file` and second one pathway abundance file. 
I will send you the corresponding file